apps/explorer/src/app/routes/parties/home/index.tsx

```diff
@@ -1,13 +1,57 @@
-import React from "react";
+import React from 'react';
+import { RouteTitle } from '../../../components/route-title';
+
+import { useNavigate } from 'react-router-dom';
+import { Routes } from '../../router-config';
+
+export const JumpToParty = () => {
```

Comment:
Our normal process is that components only used in a single route can be co-located within that route. General logic being that if a component is built for a specific case because it's neater or similar it is not generic. If another route needs a similar component it should be made generic and then hoisted to components directory.

However since the team has grown we don't have this written down anywhere and we defintiely should disucss with the wider team how we want to do this.

Reply:
Ah ok, that sounds sensible. Thanks